attributes/default.rb

```diff
@@ -136,3 +136,10 @@
 default['hadoop_spark']['spark-metrics_version']        = "#{node['scala']['version']}-2.4-1.0.5"
 default['hadoop_spark']['simpleclient_version']         = "0.8.1"
 default['hadoop_spark']['metrics-core_version']         = "3.1.5"
+
+default['hadoop_spark']['snowflake-jdbc_version']       = "3.12.17"
+default['hadoop_spark']['spark-snowflake_artifactID']   = "2.11"  
+default['hadoop_spark']['spark-snowflake_version']      = "2.8.4-spark_2.4"
+
+default['hadoop_spark']['snowflake-jdbc']['url']        = "https://repo1.maven.org/maven2/net/snowflake/snowflake-jdbc/#{node['hadoop_spark']['snowflake-jdbc_version']}/snowflake-jdbc-#{node['hadoop_spark']['snowflake-jdbc_version']}.jar"
+default['hadoop_spark']['spark-snowflake']['url']       = "https://search.maven.org/remotecontent?filepath=net/snowflake/spark-snowflake_#{node['hadoop_spark']['spark-snowflake_artifactID']}/#{node['hadoop_spark']['spark-snowflake_version']}/spark-snowflake_#{node['hadoop_spark']['spark-snowflake_artifactID']}-#{node['hadoop_spark']['spark-snowflake_version']}.jar"
```

Comment:
No you can't do that. You need to download them from repo.hops.works. Use `node['download_url']` as base